### PR TITLE
Run tests against Docker 1.7 RC2

### PR DESCRIPTION
**Pull request by aanand**
There are a few test regressions (not application regressions, thankfully). I've fixed some but not all.

### CODE REVIEW

File: compose/container.py [hunk 1]
```diff
@@ -115,6 +115,14 @@ def environment(self):
     def is_running(self):
         return self.get('State.Running')
 
+    @property
+    def cpu_set(self):
+        return self.get('HostConfig.CpusetCpus') or self.get('Config.Cpuset')
```
Comment (bfirsh) [inline compose/container.py]: Is this a bug in the engine? Shouldn't the API should remain stable?

**Comment by dnephin** [comment]
The kill tests are failing on jenkins the same way they were for me in #759 

Here's what I did to get it working on my laptop: https://github.com/docker/compose/compare/docker:master...dnephin:broken_kill_integration_tests

It does change the nature of the tests though

File: compose/container.py (hunk 1 above)
Comment (aanand) [inline compose/container.py]: Yes. Reported in https://github.com/docker/docker/issues/13663 and reverted my workarounds.

**Comment by aanand** [comment]
@dnephin Seems to be a Docker 1.7 regression - whatever signal I send to the container (even, say, SIGWINCH), it exits. I'm investigating.

**Comment by aanand** [comment]
Opened https://github.com/docker/docker/issues/13665 for the two kill test failures.

So, of the five failures:

- Two are caused by https://github.com/docker/docker/issues/13665
- Two are caused by https://github.com/docker/docker/issues/13663
- One is fixed by https://github.com/docker/docker/pull/13535, which should land in 1.7 RC2

**Comment by aanand** [comment]
Update: docker/docker#13665 is fixed by docker/docker#13666. Two bugs down, one to go.

**Comment by runcom** [comment]
Will take care of the last one really soon :)

**Comment by runcom** [comment]
all good now, last bug was fixed @aanand

**Comment by aanand** [comment]
Amazing, thanks. I've updated it to run against Docker master to confirm.

So that the build remains stable, though, I'll wait for RC2 and update it to run against that before merging.

**Comment by vdemeester** [comment]
Ok tried out with 1.7.0-rc2 and it is working :wink: !

**Comment by aanand** [comment]
All green against RC2 .

**Comment by aanand** [comment]
Rebased.

**Comment by dnephin** [comment]
LGTM

**Comment by thaJeztah** [comment]
Just in time for 1.7-rc3, which will be pushed out later today :smile:

Moving to rc3 should hopefully be less problematic now that these issues have been solved.

**Pull request merged**